From b1ea84806564d9986e41b82664aeb42e900785fd Mon Sep 17 00:00:00 2001
From: Stefan Schmidt <s.schmidt@samsung.com>
Date: Thu, 18 Dec 2014 08:58:52 +0100
Subject: [PATCH] modules: Make sure we have all linker flags setup for our
 modules.

We missed this here and test_mapo wanted eina_stringshare. Thanks to Andreas
Metzler for the report and initial patch.

@fix

Fix T1920
---
 src/modules/test_entry/Makefile.am | 2 +-
 src/modules/test_map/Makefile.am   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/modules/test_entry/Makefile.am b/src/modules/test_entry/Makefile.am
index 7393f97..a791506 100644
--- a/src/modules/test_entry/Makefile.am
+++ b/src/modules/test_entry/Makefile.am
@@ -17,6 +17,6 @@ pkg_LTLIBRARIES = module.la
 
 module_la_SOURCES = mod.c
 
-module_la_LIBADD = $(top_builddir)/src/lib/libelementary.la
+module_la_LIBADD = @ELEMENTARY_LIBS@ $(top_builddir)/src/lib/libelementary.la
 module_la_LDFLAGS = -module @EFL_LTMODULE_FLAGS@
 module_la_LIBTOOLFLAGS = --tag=disable-static
diff --git a/src/modules/test_map/Makefile.am b/src/modules/test_map/Makefile.am
index cf52653..14e7cc2 100644
--- a/src/modules/test_map/Makefile.am
+++ b/src/modules/test_map/Makefile.am
@@ -17,6 +17,6 @@ pkg_LTLIBRARIES = module.la
 
 module_la_SOURCES = mod.c
 
-module_la_LIBADD = $(top_builddir)/src/lib/libelementary.la
+module_la_LIBADD = @ELEMENTARY_LIBS@ $(top_builddir)/src/lib/libelementary.la
 module_la_LDFLAGS = -module @EFL_LTMODULE_FLAGS@
 module_la_LIBTOOLFLAGS = --tag=disable-static
-- 
2.1.3

